deploy/helm/sumologic/templates/cleanup/job.yaml

```diff
@@ -70,5 +70,5 @@ spec:
             value: "{{ template "terraform.secret.name" }}"
           {{- include "proxy-env-variables" . | nindent 10 }}
       securityContext:
-        runAsUser: 999
+        runAsUser: 1000
 {{- end }}
```

Comment:
What happened here?

Reply:
uid changed: https://github.com/SumoLogic/sumologic-kubernetes-setup/pull/104#issue-2318453310

I decided to use `1000` as `999` is already used by debian